NEWS

```diff
@@ -48,6 +48,12 @@ CHANGES WITH 253 in spe:
         * systemd-fstab-generator now supports x-systemd.makefs option for
           /sysroot (in the initrd).
 
+        * New Network param SocketPriority= to assign the Linux socket priority
+          used by DHCPv4 raw socket. Can be used in conjuntion with Link
+          VLAN EgressQosMappings to send the desired ethernet 802.1Q frame
```

Comment:
Maybe better to mention which can be configured through `EgressQosMaps=` setting in `[VLAN]` section of .netdev file.